chat.go

```diff
@@ -151,7 +171,11 @@ func processStream(stream *ssestream.Stream[openai.ChatCompletionChunk], printTh
 	}
```

Comment:
The panic on line 170 results in messy error messages with the stack trace. Please change to stderr and non-zero exit.